assets/css/nav/_bottom_nav_mobile.scss

```diff
@@ -0,0 +1,59 @@
+.m-bottom-nav-mobile {
+  background-color: $color-gray-50;
+  border-top: 1px solid $color-gray-300;
+  padding: 0.5rem 0.75rem 0.5rem 0.75rem;
+}
+
+.m-bottom-nav-mobile__links {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.m-bottom-nav-mobile__link,
+.m-bottom-nav-mobile__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 0.75rem;
+  border-radius: 0.375rem;
+  font-weight: 400;
+  @include font-body;
+  color: $color-gray-600;
+  text-decoration: none;
+  height: 2.25rem;
+  width: 2.25rem;
+
+  &:hover {
+    background-color: $color-gray-100;
+    color: $color-gray-700;
+    font-weight: 500;
+  }
+
+  svg {
+    width: 1.5rem;
+    height: 1.5rem;
+    fill: $color-gray-600;
+    stroke: none;
+  }
+
+  &.m-bottom-nav-mobile__link--active {
+    background-color: $color-eggplant-100;
+    font-weight: 500;
+    color: $color-eggplant-800;
+
+    svg {
+      fill: $color-eggplant-800;
+    }
+  }
+}
+
+.m-bottom-nav-mobile__icon {
+  &.m-bottom-nav-mobile__icon--swings-view svg {
+    transform: scaleX(-1);
```

Comment:
Out of scope for this ticket (and I know I'm the one who introduced this with the desktop left nav), but when we go and redo a bunch of our SVG handling we should definitely just replace the swings SVG with one that's pointing the right way around.